src/NuGet.Services.ServiceBus/TopicClientWrapper.cs

```diff
@@ -38,6 +38,28 @@ public TopicClientWrapper(string clientId, string clientSecret, string tenantId,
         }
 
         public Task SendAsync(IBrokeredMessage message)
+        {
+            BrokeredMessage innerMessage = GetBrokeredMessage(message);
+            return _client.SendAsync(innerMessage);
+        }
+
+        public void Send(IBrokeredMessage message)
+        {
+            BrokeredMessage innerMessage = GetBrokeredMessage(message);
+            _client.Send(innerMessage);
+        }
+
+        public void CloseSync()
+        {
+            _client.Close();
+        }
+
+        public Task Close()
+        {
+            return _client.CloseAsync();
+        }
```

Comment:
Oops this was a mistake by us. This method should've had an `Async` suffix and `await` the task. See: https://github.com/davidfowl/AspNetCoreDiagnosticScenarios/blob/master/AsyncGuidance.md#prefer-asyncawait-over-directly-returning-task

Would you mind fixing that as part of your change?

```suggestion
        public async Task CloseAsync()
        {
            await _client.CloseAsync();
        }
```

Reply:
Also, this well you rename `CloseSync` to `Close` to match the underlying SDK :)